test/tokenize.test.ts

```diff
@@ -75,7 +80,7 @@ describe('Tokenize', () => {
         card: {
           id: chance.guid(),
           type: 'card',
-          mask: {
+          data: {
```

Comment:
not something to address in this PR. but I see integration tests as higher quality tests for these SDKS

Reply:
definitely, we should get rid of these and do the same we are doing with go and python on both js and .net sdks